README.md

```diff
@@ -468,12 +482,7 @@ spec:
     key:                # string
 
 status:
-  conditions:           # []Condition containing at least one entry for `Ready`
-  - type:               # string
-    status:             # string
-    lastTransitionTime: # Time
-    reason:             # string
-    message:            # string
+  conditions:           # []metav1.Condition containing at least one entry for `Ready`
```

Comment:
I think this can be reverted. The schema is defined in the KEP-1623 which is mentioned in the normative text above. This schema can be considered as informational.

Reply:
```suggestion
  conditions:           # []Condition containing at least one entry for `Ready`
```